icu4j/tools/build/src/main/java/com/ibm/icu/dev/tool/docs/Deprecator.java

```diff
@@ -161,11 +161,9 @@ void process(File srcDir, File dstDir) {
         }
     }
 
-    /*
- @ deprecated
- *** @deprecated
- ** ** ** @deprecated
-    */
+    // @ deprecated
+    // *** @deprecated
+    // ** ** ** @deprecated
```

Comment:
```suggestion
```

Either make this a javadoc comment (`/** ... */`), or remove.
As it is they do nothing.
`@deprecated` is ignored in a regular comment.